src/etl/etl.py

```diff
@@ -1,66 +1,85 @@
+            self.logger.warning("WARNING: Test data load enabled.")
             time.sleep(1)
-            self.testObject = TestObject(True)
+            self.test_object = TestObject(True)
         else:
-            self.testObject = TestObject(False)
+            self.test_object = TestObject(False)
+
 
     def run_etl(self):
+        '''Run ETL'''
+
         self._load_and_process_data()
 
+
+    @staticmethod
     def wait_for_threads(thread_pool, queue=None):
-        logger.debug("Waiting for Threads to finish: %s" % len(thread_pool))
+        '''Wait for Threads'''
+
+        ETL.logger.debug("Waiting for Threads to finish: %s", len(thread_pool))
 
         while len(thread_pool) > 0:
-            logger.debug("Checking Threads: %s" % len(thread_pool))
+            ETL.logger.debug("Checking Threads: %s", len(thread_pool))
             for (index, thread) in enumerate(thread_pool):
-                logger.debug("Thread Alive: %s Exitcode: %s" % (thread.is_alive(), thread.exitcode))
+                ETL.logger.debug("Thread Alive: %s Exitcode: %s",
+                                 thread.is_alive(),
+                                 thread.exitcode)
                 if (thread.exitcode is None or thread.exitcode == 0) and not thread.is_alive():
-                    logger.debug("Thread Finished Removing from pool: ")
+                    ETL.logger.debug("Thread Finished Removing from pool: ")
                     thread.join()
                     del thread_pool[index]
                 elif thread.exitcode is not None and thread.exitcode != 0:
-                    logger.debug("Thread has Problems Killing Children: ")
+                    ETL.logger.debug("Thread has Problems Killing Children: ")
                     for thread1 in thread_pool:
                         thread1.terminate()
                     sys.exit(-1)
                 else:
                     pass
 
             if queue is not None:
-                logger.debug("Queue Size: %s" % queue.qsize())
+                ETL.logger.debug("Queue Size: %s", queue.qsize())
                 if queue.empty():
                     queue.join()
                     return
 
             time.sleep(5)
 
+
     def process_query_params(self, query_list_with_params):
+        '''Process Query Params'''
+
         # generators = list of yielded lists from parser
         # query_list_with_parms = list of queries, each with batch size and CSV file name.
         query_and_file_names = []
 
         for query_params in query_list_with_params:
-            cypher_query_template = query_params.pop(0)  # Remove the first query + batch size + CSV file name
+            # Remove the first query + batch size + CSV file name
+            cypher_query = query_params.pop(0)
+
             #  from the list. Format the query with all remaining paramenters.
-            query_to_run = cypher_query_template % tuple(query_params)
+            query_to_run = cypher_query % tuple(query_params)
```

Comment:
For all Cypher queries I took out the word template. Some had the word template some had the word query. Some had query_template. Some did not have either _query or _template. I chose to standardize with _query. I don't see that added knowledge to having both. It would make it longer to have it be _query_tempalate. Some places the length does matter and some places it does not.

Reply:
The issue is that they are not queries, they are query templates. This is where the templates get turned into real queries.